avalon/mongodb.py

```diff
@@ -390,17 +390,26 @@ def projects(self, query_filter=None, projection=None):
 
         find_args = [query_filter]
         if projection:
-            find_args.append(projection)
+            _projection = {"data.archived": 1}
```

Comment:
Why to create new dictionary and `update` for single key change?
```
projection["data.archived"] = True # or 1
```

Reply:
Thought that we better not change the input projection since it could be reused in outer scope.

Reply:
Very true...